### PR TITLE
totem-pl-parser: new package

### PR DESCRIPTION
**Pull request by flyn-org**
Signed-off-by: W. Michael Petullo <mike@flyn.org>

Maintainer: me
Compile tested: x86_64 master
Run tested: x86_64 master

Description:

### CODE REVIEW

File: libs/totem-pl-parser/Makefile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+PKG_SOURCE_URL:=https://download.gnome.org/sources/totem-pl-parser/3.26/
+PKG_HASH:=c0df0f68d5cf9d7da43c81c7f13f11158358368f98c22d47722f3bd04bd3ac1c
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+PKG_LICENSE:=LGPL-2-or-later
+PKG_LICENSE_FILES:=COPYING.LIB
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
```
Comment (neheb) [inline libs/totem-pl-parser/Makefile:22]: Not needed.

File: libs/totem-pl-parser/Makefile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+PKG_SOURCE:=totem-pl-parser-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://download.gnome.org/sources/totem-pl-parser/3.26/
+PKG_HASH:=c0df0f68d5cf9d7da43c81c7f13f11158358368f98c22d47722f3bd04bd3ac1c
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+PKG_LICENSE:=LGPL-2-or-later
+PKG_LICENSE_FILES:=COPYING.LIB
+
+PKG_FIXUP:=autoreconf
```
Comment (neheb) [inline libs/totem-pl-parser/Makefile:20]: Not needed.

File: libs/totem-pl-parser/Makefile (hunk 1 above)
Comment (flyn-org) [reply libs/totem-pl-parser/Makefile]: Removed `PKG_BUILD_PARALLEL:=1`.

File: libs/totem-pl-parser/Makefile (hunk 2 above)
Comment (flyn-org) [reply libs/totem-pl-parser/Makefile]: Removed `PKG_FIXUP:=autoreconf`.

**Comment by flyn-org** [comment]
Force pushed with recommended changes.

**Comment by BKPepe** [comment]
I've checked builbot and I see that this package is failing: https://downloads.openwrt.org/snapshots/faillogs/aarch64_cortex-a72/packages/totem-pl-parser/compile.txt

Snip:

```
3.26.6/ipkg-install/usr/include/totem-pl-parser/1/plparser
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/plparse/totem-pl-playlist.h to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/include/totem-pl-parser/1/plparser
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/plparse/totem-pl-parser-mini.h to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/include/totem-pl-parser/1/plparser
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/openwrt-build/plparse/totem-pl-parser-features.h to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/include/totem-pl-parser/1/plparser
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/plparse/README-videosite-script.md to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/libexec/totem-pl-parser
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/openwrt-build/totem-plparser.pc to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/lib/pkgconfig
Installing /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/openwrt-build/totem-plparser-mini.pc to /builder/shared-workdir/build/sdk/build_dir/target-aarch64_cortex-a72_musl/totem-pl-parser-3.26.6/ipkg-install/usr/lib/pkgconfig
Package totem-pl-parser is missing dependencies for the following libraries:
libgcrypt.so.20
```

**Pull request merged**